neo4j/tests/test_neo4j.py

```diff
@@ -3,38 +3,14 @@
 from datadog_checks.base.constants import ServiceCheck
 from datadog_checks.neo4j import Neo4jCheck
 
-from .common import METRICS_URL, NEO4J_VERSION
+from .common import METRICS_URL
 
 pytestmark = [pytest.mark.integration, pytest.mark.usefixtures('dd_environment')]
 
 
-def test(aggregator, dd_run_check, instance):
+def test_v4(aggregator, dd_run_check, instance):
     check = Neo4jCheck('neo4j', {}, [instance])
     dd_run_check(check)
 
     aggregator.assert_service_check('neo4j.openmetrics.health', ServiceCheck.OK)
-
-    if NEO4J_VERSION.startswith('3.5'):
-        aggregator.assert_metric('neo4j.page_cache.hits.count', tags=['db_name:global', f'endpoint:{METRICS_URL}'])
-    elif NEO4J_VERSION.startswith('4.0'):
-        aggregator.assert_metric('neo4j.page_cache.hits.count', tags=['db_name:global', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:neo4j', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:system', f'endpoint:{METRICS_URL}'])
-    elif NEO4J_VERSION.startswith('4.1'):
-        aggregator.assert_metric('neo4j.page_cache.hits.count', tags=['db_name:global', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:neo4j', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:system', f'endpoint:{METRICS_URL}'])
-    elif NEO4J_VERSION.startswith('4.2'):
-        aggregator.assert_metric('neo4j.page_cache.hits.count', tags=['db_name:global', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:neo4j', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:system', f'endpoint:{METRICS_URL}'])
-    elif NEO4J_VERSION.startswith('4.3'):
-        aggregator.assert_metric('neo4j.page_cache.hits.count', tags=['db_name:global', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:neo4j', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:system', f'endpoint:{METRICS_URL}'])
-    elif NEO4J_VERSION.startswith('4.4'):
-        aggregator.assert_metric('neo4j.page_cache.hits.count', tags=['db_name:global', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:neo4j', f'endpoint:{METRICS_URL}'])
-        aggregator.assert_metric('neo4j.check_point.duration', tags=['db_name:system', f'endpoint:{METRICS_URL}'])
-    else:
-        raise Exception(f'unknown neo4j_version: {NEO4J_VERSION}')
+    aggregator.assert_metric('neo4j.page_cache.hits.count', tags=['db_name:global', f'endpoint:{METRICS_URL}'])
```

Comment:
This specific metric isn't documented in the metadata.csv. There's a similar one that is a gauge? Do both get exposed?

Reply:
that looks like an old review. That file was recently changed and that metric is documented in the metadata.csv as both a Neo4j 4 and 5

neo4j.dbms.page_cache.hits,count,,,,The total number of page hits happened in the page cache. (Neo4j 4 and Neo4j 5),0,neo4j,page cache hit ratio,